ddev/src/ddev/e2e/agent/docker.py

```diff
@@ -93,6 +96,7 @@ def start(self, *, agent_build: str, local_packages: dict[Path, str], env_vars:
                 and 'py' not in agent_build
                 and agent_build != 'datadog/agent:6'
                 and agent_build != 'datadog/agent:7'
+                and not re.match(RELEASE_REGEX, agent_build)
```

Comment:
Can you condense this to one regular expression pattern match?

Reply:
I tried. Let me know what you think.